partner_users_controller.rb

```diff
@@ -0,0 +1,59 @@
+class V1::PartnerUsersController < V1::BaseController
+
+  before_filter(only: :create) { |c| c.check_params(:partner_account_id, :partner_user) }
+	before_filter(only: :update) { |c| c.check_params(:id, :partner_account_id, :partner_user) }
+  before_filter(only: :destroy)    { |c| c.check_params(:id, :partner_account_id) }
+  before_filter(only: :send_email) { |c| c.check_params(:id, :partner_account_id, :keyword) }
+  before_filter :require_partner_account
+  before_filter :require_user, except: :create
+
+  # POST /v1/partner_users/
+  # required parametets - partner_account_id, partner_user
+  def create
+    params[:partner_user][:crypted_password] = convert_to_md5(params[:partner_user][:crypted_password]) unless params[:partner_user][:crypted_password].blank?
```

Comment:
Identical code found in partner_users_controller.rb:13, partner_users_controller.rb:22 (mass = 72)